lib/rules/require-early-return.js

```diff
@@ -0,0 +1,43 @@
+var assert = require('assert');
+
+module.exports = function() {};
+
+module.exports.prototype = {
+    configure: function(options) {
+        assert(
+            options === 'noElse',
```

Comment:
I thought it was wanted some stricted behaviour, so this is to be open to add other options (eg 'strict'). Happy to change to be a boolean value.